tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -1,6 +1,3 @@
-! This configuration is supposed to match the rrfs_dev1 parallel, so
-! do not change the namelist without checking the current parallel.
```

Comment:
Did you check the current rrfs_dev1 parallel before removing the warning that told you to check the current rrfs_dev parallel?